### PR TITLE
Install Guide corrections

### PR DESCRIPTION
**Pull request by oliverguenther**
Bumps ruby to 2.1.7 and uses the Rails 4 notation for rake/rails

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://oliverguenther-bump-ruby-install-guide.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://oliverguenther-bump-ruby-install-guide.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: doc/operation_guides/manual/installation-guide.md [hunk 1]
```diff
@@ -219,16 +219,16 @@ prevents you from such errors.
 
 ```bash
 [openproject@host] cd ~/openproject
-[openproject@host] bundle exec rake db:create:all
-[openproject@host] bundle exec rake generate_secret_token
-[openproject@host] RAILS_ENV="production" bundle exec rake db:migrate
-[openproject@host] RAILS_ENV="production" bundle exec rake db:seed
-[openproject@host] RAILS_ENV="production" bundle exec rake assets:precompile
+[openproject@host] ./bin/rake db:create:all
```
Comment (basiszwo) [inline doc/operation_guides/manual/installation-guide.md]: could be `[openproject@host] RAILS_ENV="production" ./bin/rake db:create`

Comment (oliverguenther) [inline doc/operation_guides/manual/installation-guide.md]: Done

File: doc/operation_guides/manual/installation-guide.md [hunk 2]
```diff
@@ -219,16 +219,16 @@ prevents you from such errors.
 
 ```bash
 [openproject@host] cd ~/openproject
-[openproject@host] bundle exec rake db:create:all
-[openproject@host] bundle exec rake generate_secret_token
-[openproject@host] RAILS_ENV="production" bundle exec rake db:migrate
-[openproject@host] RAILS_ENV="production" bundle exec rake db:seed
-[openproject@host] RAILS_ENV="production" bundle exec rake assets:precompile
+[openproject@host] ./bin/rake db:create
+[openproject@host] ./bin/rake generate_secret_token
```
Comment (basiszwo) [inline doc/operation_guides/manual/installation-guide.md]: We should set a proper environment variable `echo "SECRET_KEY_BASE=`bundle exec rake secret`" >> /etc/environment`

Comment (basiszwo) [inline doc/operation_guides/manual/installation-guide.md]: bundle exec rake secret
needs to be written in backticks

**Comment by oliverguenther** [comment]
Also related to https://community.openproject.org/work_packages/21723

**Comment by basiszwo** [comment]
Merging this. be aware that this is documentation for the current dev branch and the upcoming version 5.0.

**Pull request merged**